lib/js_locale_helper.rb

```diff
@@ -156,13 +156,18 @@ def self.output_locale(locale)
     result << File.read("#{Rails.root}/vendor/assets/javascripts/moment.js")
     result << File.read("#{Rails.root}/vendor/assets/javascripts/moment-timezone-with-data.js")
     result << moment_locale(locale_str)
+    result << moment_locale(locale_str, true) # load timezones name locales
```

Comment:
The fact that you feel the need to add a comment is a good indicator that the method signature isn't clear enough. It would be a good case for a keyword argument.

```ruby
def self.find_moment_locale(locale_chain, timezone_names: false)
```

Reply:
Thanks, that's much better.